.github/workflows/publish-package.yml

```diff
@@ -36,9 +36,10 @@ jobs:
           yarn install --immutable --check-cache
           yarn build
           yarn pack
+        if: ${{ steps.release.outputs.release_created }}
 
       - name: Publish
-        run: yarn npm publish --access public
+        run: yarn publish --access public
         env:
-          NPM_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
Is this correct to change?

Reply:
https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages#publishing-packages-to-the-npm-registry